extension/oauth2clientauthextension/extension_test.go

```diff
@@ -47,11 +47,12 @@ func TestOAuthClientSettings(t *testing.T) {
 		{
 			name: "all_valid_settings",
 			settings: &Config{
-				ClientID:     "testclientid",
-				ClientSecret: "testsecret",
-				TokenURL:     "https://example.com/v1/token",
-				Scopes:       []string{"resource.read"},
-				Timeout:      2,
+				ClientID:       "testclientid",
+				ClientSecret:   "testsecret",
+				EndpointParams: url.Values{"audience": []string{"someaudience"}},
```

Comment:
Can you please add the equivalent assert to test the `EndpointParams` to complete  the test - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/549435ea23e9af63abe56945a86a0c34a1c5bba7/extension/oauth2clientauthextension/extension_test.go#L135